### PR TITLE
Update component to client in public visibility

### PR DESCRIPTION
**Pull request by udyogicx**

### CODE REVIEW

File: en/docs/develop/components/service.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Choreo defines endpoints by combining port binding, protocol, endpoint name, net
...
 | Type | The communication protocol used by the endpoint (e.g., HTTP, HTTPS, gRPC, etc.). |
-| Network Visibility | The level of visibility granted to the endpoint. Determines the level of visibility granted to the endpoint, with the following options: <ul><li>Project: the endpoint can only be accessed by other components within the same project.</li><li>Organization: the endpoint can be accessed by any component within the same organization, but not by components outside of the organization.</li><li>Public: the endpoint can be accessed by any component, regardless of its location or organization.</li></ul> |
+| Network Visibility | The level of visibility granted to the endpoint. Determines the level of visibility granted to the endpoint, with the following options: <ul><li>Project: the endpoint can only be accessed by other components within the same project.</li><li>Organization: the endpoint can be accessed by any component within the same organization, but not by components outside of the organization.</li><li>Public: the endpoint can be accessed by any client, regardless of its location or organization.</li></ul> |
```
Comment (Nashaath) [inline en/docs/develop/components/service.md:24]: Suggestion to improve clarity and avoid passive voice:
Determines the level of visibility of an endpoint. Possible values are:
- Project: Allows components within the same project to access the endpoint.
- Organization: Allows any component within the same organization to access the endpoint but restricts access to components outside the organization.
- Public: Allows any client to access the endpoint, regardless of location or organization.

**Review by Nashaath — COMMENTED** [review]
Added a minor suggestion.

File: en/docs/develop/components/service.md (hunk 1 above)
Comment (udyogicx) [reply en/docs/develop/components/service.md]: Fixed with https://github.com/wso2/docs-choreo-dev/pull/837/commits/2035102fbe0abb353e98116d176dbadfdc6a0cb5

File: en/docs/develop/components/service.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Choreo defines endpoints by combining port binding, protocol, endpoint name, net
...
 | Port | The network port on which the endpoint is accessible. |
 | Type | The communication protocol used by the endpoint (e.g., HTTP, HTTPS, gRPC, etc.). |
-| Network Visibility | The level of visibility granted to the endpoint. Determines the level of visibility granted to the endpoint, with the following options: <ul><li>Project: the endpoint can only be accessed by other components within the same project.</li><li>Organization: the endpoint can be accessed by any component within the same organization, but not by components outside of the organization.</li><li>Public: the endpoint can be accessed by any component, regardless of its location or organization.</li></ul> |
+| Network Visibility | Determines the level of visibility of an endpoint. Possible values are: <ul><li>Project: Allows components within the same project to access the endpoint.</li><li>Organization: Allows any component within the same organization to access the endpoint but restricts access to components outside the organization.</li><li>Public: Allows any client to access the endpoint, regardless of location or organization.</li></ul> |
```
Comment (Nashaath) [inline en/docs/develop/components/service.md:24]:
```suggestion
| Network Visibility | Determines the level of visibility of an endpoint. Possible values are: <ul><li>Project: Allows components within the same project to access the endpoint.</li><li>Organization: Allows any component within the same organization to access the endpoint but restricts access to components outside the organization.</li><li>Public: Allows any component to access the endpoint, regardless of location or organization.</li></ul> |
```

**Review by Nashaath — APPROVED** [review]

**Pull request merged**